recipes/sqlpp11/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 2.8.12)
+
+project(PackageTest)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
This is only available in CMake 3.1+

Please use 

`set_property(TARGET ${PROJECT_NAME} PROPERTY CXX_STANDARD 11)` instead

Reply:
Really? 3.1 has been released in 2014, what is the lowest version we have to support?

Reply:
I had to use `CMAKE_CXX_STANDARD` in another PR because `set_property` did not work (I don't remember which PR it was)

Reply:
set_property should work because it's supported since CMake 2.8. There is a new version which is set_target_properties, but it's Cmake 3.x. Anyway, prefer set_property instead of set(). set() is evil

Reply:
Well in this case `CMAKE_*`-like vars are designed to be used with `set()`, but ok I'll use the antique version

Reply:
I didn't see that `cmake_minimum_required` was called with 2.8.12... I never update test_package CMakeFiles 😅

Reply:
I think that the oldest thing we should care about is RHEL6/CentOS6 which still has CMake 2.8 per default. Unfortunately, even RHEL7/CentOS7 has 2.8 per default.

Anyways, our own CMake files aren't that complicated so we should be fine with that limitation mostly. If the upstream project we package uses a newer CMake version anyway, than obviously it doesn't matter if we use in our own wrappers something newer as well. As the package couldn't be compiled with CMake 2.8 anyway